tough/src/lib.rs

```diff
@@ -289,6 +291,10 @@ impl<'a, T: Transport> Repository<'a, T> {
         &self.timestamp
     }
 
+    pub fn delegated_targets(&self) -> Vec<&Target> {
```

Comment:
I think this function should probably be renamed to `all_targets()` or something similar since it returns all the targets, not just targets that have been delegated.